docs/data/date-pickers/validation/validation.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Date and Time pickers - Validation
+---
+
+# Date and Time pickers - Validation
+
+<p class="description">Add custom validation to user inputs.</p>
+
+All the date and time pickers have an API for adding validation constraints.
+By default they provide visual feedback if the component value doesn't meet the validation criteria.
+
+:::info
+The validation props are showcased for each type of picker component (`TimePicker`, `DatePicker`, `DateRangePicker`, etc).
+But the same props are available on all component variants.
+
+For example—the validation props available on the `DatePicker` component are also available on:
+
+- `DesktopDatePicker`
+- `MobileDatePicker`
+- `StaticDatePicker`
+
+:::
+
+## Invalid values feedback
+
+On the input—it enables its error state.
+
+{{"demo": "ValidationBehaviorInput.js", "defaultCodeOpen": false}}
+
+On the calendar and clock views—the invalid values are displayed as disabled to prevent their selection.
+
+{{"demo": "ValidationBehaviorView.js", "defaultCodeOpen": false}}
+
+## Past and future validation
+
+All pickers support the past and future validation.
+
+The `disablePast` prop prevents the selection of all values before today.
```

Comment:
Maybe we can detail more ?

```suggestion
The `disablePast` prop prevents the selection of all values before the current date and time.
For date pickers it will prevent selections of all values before today.
To time pickers, it will prevent selections of all values before the current time.
For date time pickers, it will combine both.
```

Reply:
That is a proper wall of text, but I reckon it's better to be as precise as we can—sounds good 🤔 👍

Reply:
A shorter version:

```suggestion
The `disablePast` prop prevents the selection of all values before today for date pickers and the selection of all values before the current time for time pickers.
For date time pickers, it will combine both.
```

Reply:
What do you think about using "now"?

```suggestion
The `disablePast` prop prevents the selection of all values before now. That means before today on date pickers, or before the current time, on time pickers.
```